### PR TITLE
Make use of the new logger module

### PR DESCRIPTION
**Pull request by rmohr**
Logging works now a little bit different than before, so we have to
adjust the final message which needs to be printed.

### CODE REVIEW

File: pkg/virt-controller/watch/watch.go [hunk 1]
```diff
@@ -1,121 +0,0 @@
-package watch
```
Comment (rmohr) [inline pkg/virt-controller/watch/watch.go]: Will do an extra PR for the romoval of this file

File: pkg/virt-controller/watch/vm.go [hunk 2]
```diff
@@ -145,14 +144,14 @@ func (v *vmResourceEventHandler) OnUpdate(oldObj, newObj interface{}) error {
 
 func (v *vmResourceEventHandler) OnDelete(obj interface{}) error {
 	vm := obj.(*v1.VM)
-	logger := v.logger.With("object", "VM", "action", "deleteVMPods", "name", vm.GetObjectMeta().GetName(), "UUID", vm.GetObjectMeta().GetUID())
+	logger := logging.DefaultLogger().Object(vm)
```
Comment (stu-gott) [inline pkg/virt-controller/watch/vm.go]: You might still need "action" here

**Comment by stu-gott** [comment]
Looks good!

File: pkg/virt-controller/watch/vm.go (hunk 2 above)
Comment (rmohr) [inline pkg/virt-controller/watch/vm.go]: Would give more context, that's true ...

**Pull request merged**